src/ClimaAtmos/Dycore/src/VanillaAtmosDiscretizations.jl

```diff
@@ -5,24 +5,26 @@ using ..CLIMAAtmosDycore
 AD = CLIMAAtmosDycore
 using CLIMAAtmosDycore.Grids
 using CLIMAAtmosDycore.AtmosStateArrays
+using Utilities.MoistThermodynamics
 
 export VanillaAtmosDiscretization
 
 using ParametersType
-using PlanetParameters: cp_d, cv_d, R_d, grav
+using PlanetParameters: cp_d, cv_d, R_d, grav, MSLP
 @parameter gamma_d cp_d/cv_d "Heat capcity ratio of dry air"
 @parameter gdm1 R_d/cv_d "(equivalent to gamma_d-1)"
 
 @parameter prandtl 71//10 "Prandtl number: ratio of momentum diffusivity to thermal diffusivity"
 @parameter stokes  -2//3  "scaling for viscous effect associated with volume change"
 @parameter k_μ cp_d/prandtl "thermal conductivity / dynamic viscosity"
+@parameter Cs      12//100  "Smagorinsky constant"
```

Comment:
I'm not sure where we are with this. But all of these should be adjustable parameters, collected with default values in some central place (perhaps just a namelist file for now). @charleskawczynski ?

Reply:
For now, I recommend that these parameters be passed in as a function argument via a `NamedTuple`. For example:
```
function kernel(..., params::NamedTuple)
Cs = params.Cs
...
```

If there are too many of these variables building up in the drivers then I can make a new module where these parameters can live.

Reply:
Just to note, if #186 is merged before this then these parameters can be added to the new module.